packages/strapi-admin/admin/src/containers/App/index.js

```diff
@@ -14,15 +14,21 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Switch, Route } from 'react-router-dom';
-import AdminPage from 'containers/AdminPage';
-import NotFoundPage from 'containers/NotFoundPage';
-import NotificationProvider from 'containers/NotificationProvider';
-import AppLoader from 'containers/AppLoader';
+// From strapi-helper-plugin
 import LoadingIndicatorPage from 'components/LoadingIndicatorPage';
+
 import '../../styles/main.scss';
+
+import NotFoundPage from '../NotFoundPage';
```

Comment:
Is there a lint rule to load the components in alphabetic order?

Reply:
The only rule we have is lib first, then relative path, absolute path

Reply:
Okay... it makes sense. When I saw this it was weird, especially when the propTypes, the actions, etc have to be ordered alphabetically.

Reply:
I do it manually